### DISCUSSION TITLE
Grafana Mute Time Range Error: ‘bad request data’

### DISCUSSION THREAD

**Discussion by jagan985** [grafana/grafana#73355] [Alerting]
I want to send mail alert from Grafana only by 11 AM IST and 5 PM IST. Which is 5:30 AM UTC and 11:30 AM UTC respectively. But I don’t see any specific time to mention. But I found the mute time range. I provided the mute time range as below in UTC,
IST::: 11:05 - 16:55
UTC::: 05:35 - 11:25

IST::: 17:05 - 10:55
UTC::: 11:35 - 05:25

But Grafana considers only the first-time range. If I provide the 2nd time range or add 'Another time interval', it gives error as ‘Error: bad request data’.
pls help to resolve this.

**Comment by armandgrillet** [comment]
This is an issue to investigate: #73912
